package.json

```diff
@@ -77,7 +77,10 @@
     ]
   },
   "dependencies": {
+    "@videojs/vhs-utils": "^1.2.0",
     "global": "^4.3.2",
+    "m3u8-parser": "^4.4.0",
```

Comment:
I think we may want to think about adding m3u8 parser and mpd-parser on vhs after all, as we don't want to re-include fairly big parsers twice (once in vhs and once here).